Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -2602,3 +2610,35 @@ void CLuaEngineDefs::EnginePreloadWorldArea(CVector position, std::optional<Prel
     if (option == PreloadAreaOption::ALL || option == PreloadAreaOption::COLLISIONS)
         g_pGame->GetStreaming()->LoadSceneCollision(&position);
 }
+
+
+bool CLuaEngineDefs::EngineFramerateFixingResetProperties()
+{
+    g_pMultiplayer->FramerateFixingSetPhysicsTimeStep(0);   //use default, Should we reset this when player disconnects?
+    return true;
+}
+
+bool CLuaEngineDefs::EngineFramerateFixingSetProperty(std::string strPropertyName, float timestep)
+{
+    if (strPropertyName.compare("vehicle_physics") == 0)
+    {
+        g_pMultiplayer->FramerateFixingSetPhysicsTimeStep(timestep);
+    }
+    else
+    {
+        throw std::invalid_argument("Unsupported property name at argument 1");
+    }
```

Comment:
Instead of std::string, use an enum here. Create the enum and add it to the definition in CLuaFunctionParseHelper. Then the parser will handle everything and return an error if the type is invalid.